resume-builder-ai/src/app/dashboard/optimizations/[id]/page.tsx

```
@@ -328,28 +424,31 @@ export default function OptimizationPage() {
+          <div>
+            <p className="text-sm font-medium text-blue-900 dark:text-blue-100">
+              Current Design: {currentDesignAssignment?.template?.name || 'Natural (No Design)'}
+            </p>
+            {currentDesignAssignment?.customization && (
+              <p className="text-xs text-blue-700 dark:text-blue-300 mt-1">
+                Customized • ATS Score: {currentDesignAssignment.template?.ats_score || 'N/A'}
+              </p>
+            )}
+            {!currentDesignAssignment && (
+              <p className="text-xs text-blue-700 dark:text-blue-300 mt-1">
+                Plain resume format • Click "Change Design" to apply a template
```

ESLint

[error] `"` can be escaped with `&quot;`, `&ldquo;`, `&#34;`, `&rdquo;`.

(react/no-unescaped-entities)